randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1705,26 +1705,42 @@ Extra - asset_id: collision_camera_019
   * Layers: default
   * Morph Ball Tunnel to Energy Recharge Station Secret/Tunnel to Energy Recharge Station (Gate)
   > Tile Group (WEIGHT)
-      After Ferenia - grapplepulloff1x2_000 and Lay Bomb
+      All of the following:
+          After Ferenia - grapplepulloff1x2_000
+          Lay Bomb or Lay Power Bomb
 
 > Bomb Ledge; Heals? False
   * Layers: default
   > Dock to EMMI Zone Exit Middle
       Screw Attack
   > Event - Grapple Block (grapplepulloff1x2_000)
       All of the following:
-          Morph Ball
+          Grapple Beam and Morph Ball
           Any of the following:
               Cross Bomb
               All of the following:
                   # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
-                  Bomb and Power Bombs
+                  Bomb and Main Power Bomb and Power Bombs
```

Comment:
As was said in the discord, Main Power Bomb apparently does not need to be listed

Reply:
Added it because the template "Lay Power Bomb" includes main power bomb (link: https://github.com/randovania/randovania/blob/main/randovania/games/dread/json_data/header.json?plain=1#L1413-L1421). I'll remove it for this instance. It could also be removed for the "Lay Power Bomb" template.